src/layer/marker/Marker.Drag.js

```diff
@@ -60,6 +60,8 @@ L.Handler.MarkerDrag = L.Handler.extend({
 
 		// @event movestart: Event
 		// Fired when the marker starts moving (because of dragging).
+
+		this._marker._oldLatLng = this._marker.getLatLng();
```

Comment:
I'd store the tmp variable on `this` instead of `this._marker`, in order to reduce the risk of a clash for L.Marker users.